src/modules/transaction/api/index.js

```diff
@@ -210,23 +209,27 @@ export const getTransactionFee = async ({
 
   const maxCommandFee = MODULE_COMMANDS_MAP[moduleCommand].maxFee;
   const transactionObject = fromTransactionJSON(transactionJSON, paramsSchema);
-  let numberOfEmptySignatures = 0;
 
   if (transactionJSON.moduleCommand === MODULE_COMMANDS_NAME_MAP.registerMultisignature) {
     const { optionalKeys, mandatoryKeys } = transactionJSON.params;
-    numberOfSignatures = optionalKeys.length + mandatoryKeys.length + 1;
-  } else if (wallet?.summary?.isMultisignature) {
-    numberOfEmptySignatures =
-      wallet.keys.mandatoryKeys.length + wallet.keys.optionalKeys.length - numberOfSignatures;
+    numberOfSignatures = optionalKeys.length + mandatoryKeys.length;
   }
 
   // Call API to get network specific base fees
   const baseFees = [];
-  const minFee = transactions.computeMinFee(transactionObject, paramsSchema, {
-    baseFees,
-    numberOfSignatures,
-    numberOfEmptySignatures,
-  });
+  const minFee = transactions.computeMinFee(
+    {
+      ...transactionObject,
+      params: {
+        ...transactionObject.params,
+        signatures: new Array(numberOfSignatures).fill(0).map(() => Buffer.alloc(64)),
```

Comment:
Also, I think this should not be done this way only empty signatures should be replaced with an empty buffer, this is because when members are signing, there could be empty signatures(meaning a member is yet to sign) and there could be actual signatures(meaning a member has signed). so in this case I think it can be done like this @ikem-legend 

```suggestion
        ...(numberOfSignatures && !transactionObject.params. signatures.length && { signatures: Array(numberOfSignatures).fill(Buffer.alloc(64)) }) ,
```

Reply:
@eniolam1000752 Fixed